src/client/components/meals/NewMeal.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from "react"
+
+
+export default function NewMeal({ meals, setMeals }) {
+    const [title, setTitle] = useState("")
+    const [description, setDescription] = useState("")
+    const [location, setLocation] = useState("")
+    const [when, setWhen] = useState("")
+    const [maxReservations, setMaxReservations] = useState(0)
+    const [price, setPrice] = useState(0)
```

Comment:
You could potentially use a reducer(useReducer) in a situation like this which can simplify and organize your state logic. You don't necessarily have to implement it here, it's just for the future.